Source/Schema.NET/TimeSpanToISO8601DurationValuesConverter.cs

```diff
@@ -11,6 +12,34 @@
     /// <seealso cref="JsonConverter" />
     public class TimeSpanToISO8601DurationValuesConverter : ValuesConverter
     {
+        /// <inheritdoc />
```

Comment:
This unwraps the nullable, deserializes the TimeSpan with the same XmlConverter used to serialize and wraps again in a Values<TimeSpan> or its nullable version